coreos-root-bash-profile-workaround.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Work around https://bugzilla.redhat.com/show_bug.cgi?id=1193590
```

Comment:
Minor: how about putting the RHBZ link in `Documentation=` and making the description something like "Seed default bash profile for root"?

Reply:
Done :arrow_down: